tagilmo/utils/vereya_wrapper.py

```diff
@@ -393,15 +406,34 @@ def stop(self, idx=0):
         if idx < len(self.agent_hosts):
             self.agent_hosts[idx].stop()
 
-    def getHumanInputs(self, nAgent=0):
-        if self.observe[nAgent] is not None:
-            data = self.observe[nAgent]
-            if "input_events" in data:
-                return data["input_events"]
+    def getHumanInputs(self, nAgent=None):
+        return self.getParticularObservation('input_events', nAgent)
 
     def placeBlock(self, x: int, y: int, z: int, block_name: str, placement: str, nAgent=0):
         self.agent_hosts[nAgent].sendCommand("placeBlock {} {} {} {} {}".format(x, y, z, block_name, placement))
 
+    def _sendMotionCommand(self, command, value, nAgent=None):
+        if nAgent is None: nAgent = self.nAgent
```

Comment:
Does `nAgent` correspond to the number of agents? Should it be `self.nAgent-1` or `0`? We have not been using multiple agents for quite a while. I wonder if it is still supported at all.

Reply:
nAgent is agent's number, in our tests it's always 0, i updated the code so that nAgent might be also a controlled mob's uuid

Reply:
malmo had api for having multiple connections with one agent per connection, i ported code as it was to python, it wasn't tested and might not work. Now we can have multiple entities in one connection and also a default controlled entity.